app/views/devise/registrations/edit.html.erb

```diff
@@ -1,39 +1,41 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
-
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
+<div class="form-container">
+<%= devise_error_messages! %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3>Edit <%= resource_name.to_s.humanize %></h3>
+      <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put, role: 'form'}) do |f| %>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :email %>
+        <%= f.email_field :email, autofocus: true, class: 'form-control input-lg' %>
+        <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+          <div>Currently waiting on email confirmation for: <%= resource.unconfirmed_email %></div>
+        <% end %>
+      </div>
+      <div class="form-group">
```

Comment:
this is just personal preference, but I find html easier to read & reason about with spacing between >1-line chunks - i.e.:
```

<div class="form-container">
  <%= devise_error_messages! %>
  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put, role: 'form'}) do |f| %>
    <div class="panel panel-default">
      <div class="panel-heading">
        <h3>Edit <%= resource_name.to_s.humanize %></h3>
      </div>

      <div class="panel-body">
        <div class="form-group">
          <%= f.label :email %>
          <%= f.email_field :email, autofocus: true, class: 'form-control input-lg' %>

          <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
            <div>Currently waiting on email confirmation for: <%= resource.unconfirmed_email %></div>
          <% end %>
        </div>

        <div class="form-group">
          <p><em>Leave these fields blank if you don't want to change your password.</em></p>

          <%= f.label :password %>
          <%= f.password_field :password, autocomplete: 'off', class: 'form-control input-lg' %>

          <%= f.label :password_confirmation %>
          <%= f.password_field :password_confirmation, class: 'form-control  input-lg' %>
        </div>

        <div class="form-group">
          <p><em>You must enter your current password to make changes.</em></p>

          <%= f.label :current_password %>
          <%= f.password_field :current_password, class: 'form-control input-lg' %>
        </div>

        <%= f.submit 'Update', class: 'btn btn-default btn-lg' %>      
      </div>
    </div>
  <% end %>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h3>Cancel Account</h3>
    </div>

    <div class="panel-body">
      <p>We're sad to see you go!</p>
      <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete, class: 'btn btn-default btn-lg' %>
    </div>
  </div>
</div>